conda/recipes/librmm/meta.yaml

```diff
@@ -60,12 +65,19 @@ outputs:
       run_exports:
         - {{ pin_subpackage("librmm", max_pin="x.x") }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% else %}
         - {{ compiler('cuda') }}
```

Comment:
Do we actually need to ignore run exports from the new compiler packages? Do we know what specifically this is for? The only export I see on either the [nvcc](https://github.com/conda-forge/cuda-nvcc-feedstock/blob/main/recipe/meta.yaml) or [impl](https://github.com/conda-forge/cuda-nvcc-impl-feedstock/blob/main/recipe/meta.yaml) recipes is cuda-version, which I believe should be safe. IOW I think we may only need to ignore run exports for CUDA 11 going forward.

Reply:
If I'm right then we'll also want to update other recipes.

Reply:
This is probably a good call. Fixed in a1156d6.

Reply:
We do this to make sure that `cudatoolkit` doesn't get pulled in when just using the `librmm` headers package

Reply:
I agree that we need it for CUDA 11, was specifically referring to removing the CUDA 12 branch. Bradley's changes look good to me.